src/pages/Releases/ReleasesPage.js

```diff
@@ -19,6 +18,8 @@ import Header from "../../components/Header/Header";
 import { isLatestRelease } from "../../helpers/releasesInfo";
 import { useReleasesURL } from "../../providers/releases/ReleasesURLProvider";
 
+const marked = require('marked/lib/marked');
```

Comment:
If there is no difference in the usage of `marked` then what issue is this PR solving?

Reply:
I guess its the difference between import and require, which somehow works.